src/common/ZulipStatusBar.js

```diff
@@ -66,16 +66,12 @@ class ZulipStatusBar extends PureComponent<Props> {
   }
 }
 
-export default connect((state, props) => ({
+export default connect((state: GlobalState, props: Object) => ({
```

Comment:
I don't think `Object` is a big improvement over the not-provided-type.

Reply:
(Added just to remove flowFixMe comments)

Reply:
In TopicListScreen, StreamScreen , we are already doing this. So this is the same for statusBar too.

Reply:
Removing these fixme comments is a bug, not a feature. The fixmes say that we don't have type-checking on the `narrow` property being present here. After this change... we still don't have type-checking on the `narrow` property being present, plus we don't have it on any other properties, plus we don't have the fixme comments pointing this out.

If on the other hand you were to find a way to *get* proper type-checking on the `narrow` property, then that would remove these fixme comments in the good way. :-)

Reply:
Ya! got it 👍  okay removing it. Then I guess we should remove it from `TopicListScreen, StreamScreen` too.